### DISCUSSION TITLE
Alternate stable sats denominations

### DISCUSSION THREAD

**Discussion by humansinstitute** [GaloyMoney/galoy#1402] [Q&A] [answered]
Curious if there are any options for generating stable sats in different currencies outside of USD, of particular interest to me would be AUD. 

I am assuming that given the current design this is a no, given the design as follows:   

1) Current implementation matches the USD account liability exposure.
2) This is achieved using bitcoin posted to exchange A and a trading bot to match shorts with the USD account balance in the community banking app.
3) As such only liquid currency pairs on Exchange A would be possible.

Also I'm assuming that this is somewhat modular, such that I could write a "new bot" for Exchange B (if it had the correct exchange pair). The thinking being this could inherit the same logical which matches the liability, dashboard tracking and the required short trades but attaches a different bot to execute and track the required trades. 

Just keen to understand if I've interpreted this correctly?

**Comment by nicolasburtey** [comment, marked as answer]
the main limitation to get a stablesats on AUD would indeed be that there is no derivative exchange with decent volume doing BTC/AUD

the dealer bot itself is would be replaced by another bot, our micro-services architecture make this easy.

**Reply by humansinstitute** [reply]
Makes sense. I can't see why a liquid derivatives exchange would grow in AUD any time soon unfortunately

**Reply by nicolasburtey** [reply]
an alternative would be to do a BTC/USD hedging, and another USD/UAD hedging on a traditional derivatives market. but 1/ it makes it a lot more complex - not a good idea because this increases the odds of something goes wrong 2/ I've no idea how a "traditional" (for fiat currency only) derivative market work, but I don't believe any of them would accept bitcoin as collateral

**Reply by humansinstitute** [reply]
Yep - I had considered that, but I'm with you, I think that just adds extra levels of complexity that no-one really needs.  

My current thinking is that the AUD "peg" is mostly for merchants, so it could be that users can hold USD / Sats and then there is an alternate service presented to merchants that allows them to sell bitcoin on transaction settlement and just clear that through local exchanges.
